open-craft/edx-platform
# feat: cherry pick mfe config api

## Description

This is a backport of multiple changes which add and modify the MFE config API

* https://github.com/openedx/edx-platform/pull/30473
* https://github.com/openedx/edx-platform/pull/30830

**Testing instructions in devstack**

- Run nutmeg devstack, and update your edx-platform to this MR.
- Start lms using `make lms-up`
- Open http://localhost:18000/api/mfe_config/v1/, it should return a json with mfe config.
